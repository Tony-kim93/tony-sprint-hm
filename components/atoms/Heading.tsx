import React, { memo } from 'react';

interface HeadingProps {
  text: string;
}

const Heading = memo(({ text }: HeadingProps) => <h1>{text}</h1>);

export default Heading;
