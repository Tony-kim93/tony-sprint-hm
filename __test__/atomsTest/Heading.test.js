import React from 'react';
import { render } from '@testing-library/react';
import Heading from '../../components/atoms/Heading';

describe('<HEADING>', () => {
  it('test h1 tag', () => {
    const text = 'text';
    const { getByText } = render(<Heading text={text} />);
    expect(getByText('text')).toBeInTheDocument();
    expect(getByText('text')).toMatchSnapshot();
  });
});
