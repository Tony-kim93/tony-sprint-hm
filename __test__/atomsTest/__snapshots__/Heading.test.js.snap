// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HEADING> test h1 tag 1`] = `
<h1>
  text
</h1>
`;
